composer.json

```diff
@@ -19,48 +18,45 @@
       "email": "cezary.stepkowski@protonmail.com"
     }
   ],
+  "homepage": "https://github.com/protung/easyadmin-plus-bundle",
+  "support": {
+    "issues": "https://github.com/protung/easyadmin-plus-bundle/issues",
+    "source": "https://github.com/protung/easyadmin-plus-bundle"
+  },
   "require": {
     "php": "^8.0",
     "ext-dom": "*",
-    "azjezz/psl": "^1.7.3",
-    "easycorp/easyadmin-bundle": "^3.5.7",
-    "symfony/asset": "^5.3",
-    "symfony/cache": "^5.3",
-    "symfony/config": "^5.3",
-    "symfony/dependency-injection": "^5.3",
-    "symfony/event-dispatcher": "^5.3",
-    "symfony/filesystem": "^5.3",
-    "symfony/form": "^5.3",
-    "symfony/framework-bundle": "^5.3",
-    "symfony/http-foundation": "^5.3",
-    "symfony/http-kernel": "^5.3",
-    "symfony/property-access": "^5.3",
-    "symfony/translation": "^5.3",
-    "symfony/twig-bundle": "^5.3"
+    "azjezz/psl": "^1.9.3",
+    "easycorp/easyadmin-bundle": "^4.0.4",
```

Comment:
We should probably start supporting patch versions only and explicitly upgrade when a new minor version of EasyAdmin is released.

I suggest `~4.0.4`

Reply:
I agree, because it already happened that minor changes in EasyAdmin required some updates in this library.